src/libs/workspace-utils.ts

```diff
@@ -26,7 +26,7 @@ export interface GoogleWorkspaceInfo extends BaseWorkspaceInfo {
 export type WorkspaceInfo = AzureWorkspaceInfo | GoogleWorkspaceInfo
 
 export const isGoogleWorkspaceInfo = (workspace: WorkspaceInfo): workspace is GoogleWorkspaceInfo => {
-  return !!((workspace as GoogleWorkspaceInfo).googleProject)
+  return workspace.cloudPlatform === 'Gcp'
```

Comment:
Either check is valid, but changing to verifying `cloudPlatform` for consistency with `isAzureWorkspace`.

Reply:
It actually turns out that in some Analysis usages, they only pull out and pass around select things from the Workspace details. Therefore switching this to `cloudPlatform` would require changing some of their code to include `cloudPlatform` (else renaming and copying of Analyses breaks). To minimize impact, switching back to checking `googleProject`.

Reply:
Actually, looking at the [unit tests](https://github.com/DataBiosphere/terra-ui/blob/9170b4cbf86ee32723290dcd55cc9b078284a78a/src/pages/workspaces/workspace/analysis/modals/AnalysisDuplicator.test.ts#L81) and the TS interface, cloudPlatform is expected to be in WorkspaceInfo, so I have changed the calling locations to include it. FYI @jdcanas and @n8landolt 

I am surprised that typescript wasn't complaining about WorkspaceInfo missing the `cloudPlatform`. @nawatts, do you know why?

Reply:
It looks like this PR adds `cloudPlatform` in two JS files (Analyses.js and AnalysisLauncher.js). Those files would not be checked by TypeScript.

Reply:
Oh right. I was thinking that it would be caught because `isGoogleWorkspaceInfo` is called from `AnalysisDuplicator`, which is in TypeScript, but I guess that the whole chain has to be TypeScript for the input argument to be analyzed poperly.

Reply:
Yeah, `AnalysisDuplicator`'s `workspaceInfo` prop is typed as `WorkspaceInfo`.
https://github.com/DataBiosphere/terra-ui/blob/0e178702932416959b76a04ab790270b48dbd111/src/pages/workspaces/workspace/analysis/modals/AnalysisDuplicator.ts#L35

So TS will validate that prop can be passed into `isGoogleWorkspaceInfo`.
https://github.com/DataBiosphere/terra-ui/blob/0e178702932416959b76a04ab790270b48dbd111/src/pages/workspaces/workspace/analysis/modals/AnalysisDuplicator.ts#L63

But it can't check what's passed to `AnalysisDuplicator` in `Analyses` because Analyses.js is JS not TS.
https://github.com/DataBiosphere/terra-ui/blob/0e178702932416959b76a04ab790270b48dbd111/src/pages/workspaces/workspace/analysis/Analyses.js#L474